### PR TITLE
Add admin settings for the Calendar Trash bin

### LINKED ISSUE #7696: Document calendar trashbin admin setting
> `php occ config:app:set dav calendarRetentionObligation --value=%value%` where `%value%` must be replaced by a number of seconds. 0 equals to disabled.

### PR DESCRIPTION
**Pull request by tcitworld**
Closes #7696

image

### CODE REVIEW

File: user_manual/groupware/calendar.rst [hunk 1]
```diff
@@ -225,7 +225,7 @@ You can also tell when the recurrence ends.
 Trash bin
 ~~~~~~~~~
 
-If you delete events, tasks or a calendar in Calendar, your data won't be gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period of 30 days those items will be deleted permanently. You can also permanently delete earlier if you wish.
+If you delete events, tasks or a calendar in Calendar, your data won't be gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period which defaults to 30 days (your administrator may have changed this setting), those items will be deleted permanently. You can also permanently delete earlier if you wish.
```
Comment (rakekniven) [inline user_manual/groupware/calendar.rst:228]: administrator -> administration

File: admin_manual/groupware/calendar.rst [hunk 2]
```diff
@@ -76,6 +76,19 @@ If you need to allow this, change the following parameter to::
 
  php occ config:app:set dav webcalAllowLocalAccess --value yes
 
+Trash bin
+---------
+
+Nextcloud supports a calendar, events and tasks trash bin.
+
+The default delay before objects are purged from the trash bin is of 30 days. A backgroup job runs every 6 hours to cleanup expired objects.
```
Comment (rakekniven) [inline admin_manual/groupware/calendar.rst:84]: backgroup -> background

Comment (rakekniven) [reply admin_manual/groupware/calendar.rst:84]: "is of 30 days" -> "is 30 day"

Comment (rakekniven) [reply admin_manual/groupware/calendar.rst:84]: cleanup -> clean up

File: user_manual/groupware/calendar.rst (hunk 1 above)
Comment (rakekniven) [reply user_manual/groupware/calendar.rst:228]: your data won't be gone yet. -> your data is not gone yet.

Comment (rakekniven) [reply user_manual/groupware/calendar.rst:228]: You can also permanently delete earlier if you wish -> You can also permanently delete items earlier if you wish

**Review by rakekniven — CHANGES_REQUESTED** [review]
Please see my comments and keep in mind that I am not an english native speaking person 😄

File: admin_manual/groupware/calendar.rst [hunk 3]
```diff
@@ -76,6 +76,19 @@ If you need to allow this, change the following parameter to::
 
  php occ config:app:set dav webcalAllowLocalAccess --value yes
 
+Trash bin
+---------
+
+Nextcloud supports a calendar, events and tasks trash bin.
```
Comment (GretaD) [inline admin_manual/groupware/calendar.rst:82]: @ChristophWurst when will we change the trash bin name to Deleted Items? If at all?

Comment (rakekniven) [reply admin_manual/groupware/calendar.rst:82]: Good comment. That would be align it to the Files app, there it is called "Deleted files".

File: admin_manual/groupware/calendar.rst (hunk 2 above)
Comment (Valdnet) [reply admin_manual/groupware/calendar.rst:84]:
```suggestion
The default delay before objects are purged from the trash bin is 30 days. A background job runs every 6 hours to clean up expired objects.
```

File: user_manual/groupware/calendar.rst (hunk 1 above)
Comment (Valdnet) [reply user_manual/groupware/calendar.rst:228]: To facilitate changes 😉.
```suggestion
If you delete events, tasks or a calendar in Calendar, your data is not gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period which defaults to 30 days (your administration may have changed this setting), those items will be deleted permanently. You can also permanently delete items earlier if you wish.
```

**Review by Valdnet — CHANGES_REQUESTED** [review]

File: user_manual/groupware/calendar.rst (hunk 1 above)
Comment (Valdnet) [reply user_manual/groupware/calendar.rst:228]: To facilitate changes 😉.
```suggestion
If you delete events, tasks or a calendar in Calendar, your data is not gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period which defaults to 30 days (your administration may have changed this setting), those items will be deleted permanently. You can also permanently delete items earlier if you wish.
```

**Review by Valdnet — CHANGES_REQUESTED** [review]

File: admin_manual/groupware/calendar.rst (hunk 3 above)
Comment (ChristophWurst) [reply admin_manual/groupware/calendar.rst:82]: We might do that but right now the UI says *trash bin*. So the documentation should be consistent. If we rename then docs should only change for 23+ or whatever versions will be affected

File: user_manual/groupware/calendar.rst (hunk 1 above)
Comment (ChristophWurst) [reply user_manual/groupware/calendar.rst:228]: > administrator -> administration

used just like that in other docs as well. this is a bigger, separate change, isn't it?

is there a glossary with terms to use / not to use?

Comment (Valdnet) [reply user_manual/groupware/calendar.rst:228]: Administrator is a person (can be one or more).
Administration is a team of administrators. Is the more official wording.

Comment (rakekniven) [reply user_manual/groupware/calendar.rst:228]: > used just like that in other docs as well. this is a bigger, separate change, isn't it?

Yes, this is a bigger change. But IMHO this is an ongoing change. We have many occurences of "admin" and "administrator". "admin" is an abbreviation and if we use "administration" we get gender neutral.

Valdnet and me correcting all new strings. Mostly in nextcloud projects and second in docs.
Please use it.

File: admin_manual/groupware/calendar.rst (hunk 3 above)
Comment (rakekniven) [reply admin_manual/groupware/calendar.rst:82]: Oh, my thought was to change this in UI as well. It is a relative new feature and therefore it is easy to change I think.
Jm2c.

**Review by rakekniven — APPROVED** [review]

**Review by miaulalala — APPROVED** [review]

**Comment by tcitworld** [comment]
Backport for stable22 in https://github.com/nextcloud/documentation/pull/7794

**Pull request merged**